servers/rendering/renderer_rd/shaders/scene_forward_lights_inc.glsl

```diff
@@ -194,7 +194,7 @@ void light_compute(vec3 N, vec3 L, vec3 V, float A, vec3 light_color, float atte
 		float D = D_GGX_anisotropic(cNdotH, ax, ay, XdotH, YdotH);
 		float G = V_GGX_anisotropic(ax, ay, dot(T, V), dot(T, L), dot(B, V), dot(B, L), cNdotV, cNdotL);
 #else // LIGHT_ANISOTROPY_USED
-		float D = D_GGX(cNdotH, alpha_ggx);
+		float D = D_GGX(cNdotH, roughness);
```

Comment:
This change isn't needed. 

In Godot, ``roughness`` is used for perceptual roughness. In Filament they distinguish between ``roughness`` and ``perceptual_roughness``. In Filament ``roughness = perceptual_roughness * perceptual_roughness`` therefore ``roughness`` in filament is the same as ``alpha_ggx`` in Godot

Reply:
Ah! I got it. Thank you. I applied it 😊😊😊😊